src/main/java/bowling/view/FrameView.java

```diff
@@ -0,0 +1,91 @@
+package bowling.view;
+
+import bowling.domain.frame.FinalFrame;
+import bowling.domain.frame.Frames;
+import bowling.domain.status.Miss;
+import bowling.domain.status.Ready;
+import bowling.domain.status.Spare;
+import bowling.domain.status.Strike;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toList;
+
+public class FrameView {
+    private static final int MAX_FRAME_NO = 10;
+    private static final int MIN_FRAME_NO = 1;
+    public static final String STRIKE_MARK = "X";
+    public static final String SPARE_MARK = "/";
+    public static final String TWO_BLANK = "  ";
+    public static final String ONE_BLANK = " ";
+    public static final String BAR_MARK = "|";
+    public static final String NULL_MARK = " ";
+    public static final String GUTTER_MARK = "-";
+
+    public static List<String> frameResultTitles() {
+        return IntStream.rangeClosed(MIN_FRAME_NO, MAX_FRAME_NO)
+                .mapToObj(i -> String.format("%02d", i))
+                .collect(toList());
+    }
+
+    public static List<String> frameResultContents(Frames frames) {
+        List<String> contents = getNormalFramesContents(frames);
+        contents.add(getFinalFrameContents(frames));
+
+        for (int i = frames.getFramesSize(); i < MAX_FRAME_NO; i++) {
+            contents.add(String.format("%7s", BAR_MARK));
+        }
+
+        return contents;
+    }
+
+    private static String getFinalFrameContents(Frames frames) {
+        String finalContent = "";
+        if (frames.getCurrentFrame().isFinalFrame()) {
```

Comment:
early return 을 활용해서 indent 를 줄어보시면 좋겠습니다. :)

```suggestion
        if (!frames.getCurrentFrame().isFinalFrame()) {
            return "";
        }
```